app/funcs/csv_reader.py

```
@@ -80,11 +80,65 @@ def read_csv(self, file: Path, **kwargs) -> Generator[Dict[int, Dict], None, Non
-                    yield rec
+                try:
+                    if file.suffix.lower() == '.csv':
+                        recs = self.read_csv(file, **kwargs)
+                    elif file.suffix.lower() == '.parquet':
+                        recs = self.read_parquet(file, **kwargs)
+                    else:
+                        continue
+                        
+                    for rec in recs:
+                        yield rec
+                except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)